docs/user_guides/templates/character_controller_slopes.mdx

```diff
@@ -26,27 +26,10 @@ character_controller.min_slope_slide_angle = 30.0.to_radians();
 <bevy>
 
 ```rust
-/* Configure the character controller when the collider is created. */
-// Snap to the ground if the vertical distance to the ground is smaller than 0.5.
-commands.spawn(Collider::ball(0.5))
-    .insert(KinematicCharacterController {
-        // Don’t allow climbing slopes larger than 45 degrees.
-        max_slope_climb_angle: 45.0.to_radians(),
-        // Automatically slide down on slopes smaller than 30 degrees.
-        min_slope_slide_angle: 30.0.to_radians(),
-        ..default()
-    });
+<load path='/2d/bevy/examples/character_controller2.rs' marker="Slopes1" />
```

Comment:
`"` should be `' ` ; TODO: add an explicit error to avoid this slipping through.

Reply:
fixed + added tests and visible errors when launching the script